arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,536 @@
+        "operationId": "RpProvision_GetAllServersForResourceGroupAsync",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/ProvisionResultList" }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/servers": {
+      "get": {
+        "description": "List all the Analysis Services servers for the given subscription",
+        "tags": [ "RpProvision" ],
+        "operationId": "RpProvision_ListAllServersForSubscriptionAsync",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/ProvisionResultList" }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/locations/{location}/operationstatuses/{operationId}": {
+      "get": {
+        "description": "Gets all the Analysis Services servers for the give resource group",
+        "tags": [ "RpProxy" ],
+        "operationId": "RpProxy_GetOperationStatusAsync",
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Identifier for the operation for which to get status"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Region in which the server operation was executed"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/locations/{location}/operationresults/{operationId}": {
+      "get": {
+        "description": "Gets result of the specified operation",
+        "tags": [ "RpProxy" ],
+        "operationId": "RpProxy_GetOperationResultAsync",
```

Comment:
how is getOperationStatus different from getOperationResult, apart from the request url. They both return "OperationDetail". As a customer when am I supposed to use these two operations. If this information is provided in the description it will really help the customer.